src/components/Request/Pay/utils.ts

```diff
@@ -10,3 +10,9 @@ export enum EPeanutLinkType {
     native,
     erc20,
 }
+
+export enum RequestScreens {
+    LOADING = 'LOADING',
+    CLAIM = 'CLAIM',
+    NOT_FOUND = 'NOT_FOUND',
+}
```

Comment:
1. Is this the same enum we have in claim screens?
2. Are these all the possible states?
3. Is RequestScreens the correct name for an enum of strings?